packages/story-editor/src/components/checklist/checks/shared/test/thumbnailWrapper.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { cleanup, render, screen } from '@testing-library/react';
+import { ThemeProvider } from 'styled-components';
+import { theme as dsTheme } from '@web-stories-wp/design-system';
+/**
+ * Internal dependencies
+ */
+import { ThumbnailWrapper } from '..';
+import { MAX_THUMBNAILS_DISPLAYED } from '../../../../checklistCard';
+
+// eslint-disable-next-line react/prop-types
+const Wrapper = ({ children }) => (
+  <ThemeProvider theme={dsTheme}>{children}</ThemeProvider>
+);
+
+describe('ThumbnailWrapper', () => {
+  afterEach(cleanup);
```

Comment:
I wonder if there's like a root level `afterEach` we could put that in. Is that a thing? or are they always within the scope of the describe statement?